app/brave_settings_strings.grdp

```diff
@@ -216,6 +216,32 @@
   <message name="IDS_SETTINGS_APPEARANCE_SETTINGS_CONTENT_SECTION" desc="The label for section containing settings related to content">
     Content
   </message>
+  <if expr="enable_containers">
+    <message name="IDS_SETTINGS_APPEARANCE_SETTINGS_CONTAINERS_SECTION" desc="The label for section containing settings related to containers">
+      Containers
+    </message>
+    <message name="IDS_SETTINGS_APPEARANCE_SETTINGS_CONTAINERS_DESC" desc="The label for section containing settings related to containers">
+      Organize your browsing with containers. <ph name="BEGIN_LINK">&lt;a target="_blank" href="$1"&gt;</ph>Learn more<ph name="END_LINK">&lt;/a&gt;</ph>
```

Comment:
cc @rebron to refine all these strings. For details where this is actually visible pls see attached video in the PR description.